lib/mix/test/mix/tasks/compile.erlang_test.exs

```diff
@@ -16,7 +16,7 @@ defmodule Mix.Tasks.Compile.ErlangTest do
   @tag erlc_options: [{:d, ~c"foo", ~c"bar"}]
   test "raises on invalid erlc_options" do
     in_fixture("compile_erlang", fn ->
-      assert_raise Mix.Error, ~r"Compiling Erlang file '.*' failed", fn ->
+      assert_raise Mix.Error, ~r/Compiling Erlang file ~c".*" failed/, fn ->
```

Comment:
In this case, I think we should convert the file to a string and have it show up as `"..."`. WDYT?

Reply:
Makes sense 👍

Reply:
Fixed in https://github.com/elixir-lang/elixir/pull/12074/commits/55a2a4ef1ba203fc6e7ec130329abb56f2fd26cc